internal/node/bazel_require_script.js

```diff
@@ -1,6 +1,14 @@
 // Adapt node programs to run under Bazel
 // Meant to be run in a --require hook
 
+if(global.BAZEL_NODE_PATCHES) {
```

Comment:
I'm confused why this file is still here. Is the new package not wired up yet?

Reply:
my intention was to land just the package and then wire it in. I have the wiring done and tested locally. 

Turns out waiting is not really useful so i rebased my branch to master which includes the bazel_require_script.js and i'll add it to the pr.

without any other guidance the internal node rule will depend on the file group and the appropriate files will be included and ran in the bazel_node_require_script.js

Reply:
a last issue here is i need to figure out what the file paths will be when i depend on files in node-patches

Reply:
let's remove the wiring from this PR since it isn't tested yet